lib/libc/stdlib/malloc/mrs/mrs.c

```diff
@@ -1391,6 +1426,9 @@ mrs_real_malloc(size_t size)
 static void *
 mrs_malloc(size_t size)
 {
+	if (__predict_false(!mrs_initialized))
```

Comment:
This  is checked by mrs_init

Reply:
I've switched to an unconditional mrs_init(). Oddly, jemalloc does actually check the init state twice like this did.